### PR TITLE
Update copy on /gcp/pro

### PR DESCRIPTION
**Pull request by petesfrench**
## Done

- Update copy on /gcp/pro based on [copydoc](https://docs.google.com/document/d/1zR1m6CH1POU5tnzNzgNYanG2_EGQtKVNR3_dPew1pf4/edit#heading=h.pmc79wve6nh5)
- Formatting (I've commented the copy changes)

## QA

- Go to https://ubuntu-com-13816.demos.haus/gcp/pro and check the changes match the [copydoc](https://docs.google.com/document/d/1zR1m6CH1POU5tnzNzgNYanG2_EGQtKVNR3_dPew1pf4/edit#heading=h.pmc79wve6nh5)

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-10793

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)

### CODE REVIEW

File: templates/gcp/pro.html [hunk 1]
```diff
@@ -1,232 +1,333 @@
...
-      }}
+  <section class="p-strip--suru-topped is-bordered">
+    <div class="row u-equal-height">
+      <div class="col-7 col-medium-4">
+        <h1>Ubuntu Pro for Google Cloud</h1>
+        <p class="p-heading--3">For production. For professionals.</p>
+        <p>
+          Ubuntu Pro for GCP is a premium image delivering the most comprehensive security and GCP compliance features. Ubuntu Pro is suitable for small and large-scale Linux enterprise operations offering pay-as-you-go billing on your existing GCP invoice.
+        </p>
+        <h3 class="p-heading--4">Ubuntu Pro 24.04 LTS</h3>
```
Comment (petesfrench) [inline templates/gcp/pro.html:22]: changed

File: templates/gcp/pro.html [hunk 2]
```diff
@@ -1,232 +1,333 @@
...
+        <p>
+          Delivers additional layer of security and compliance services and security patches covering a wider range of packages, until 2032.
+        </p>
+        <p>
+          <a href="https://console.cloud.google.com/compute/instancesAdd?imageId=4629240575064276470&imageProjectId=ubuntu-os-cloud"
```
Comment (petesfrench) [inline templates/gcp/pro.html:27]: changed

File: templates/gcp/pro.html [hunk 3]
```diff
@@ -1,232 +1,333 @@
...
+    <div class="u-fixed-width">
+      <h2>Previous Releases</h2>
     </div>
-    <div class="col-4 p-divider__block">
-      <h3 class="p-heading--4">Ubuntu Pro 18.04 LTS</h3>
-      <p>Ubuntu 18.04 LTS release includes 5.4 kernel and provides extended maintenance through 2028.</p>
-      <p><a href="https://console.cloud.google.com/marketplace/product/ubuntu-os-pro-cloud/ubuntu-pro-bionic" class="p-button--positive"><span>Launch now</span></a></p>
+    <div class="row p-divider">
+      <div class="col-3 p-divider__block">
+        <h3 class="p-heading--4">Ubuntu Pro 22.04 LTS</h3>
```
Comment (petesfrench) [inline templates/gcp/pro.html:53]: Changed

File: templates/gcp/pro.html [hunk 4]
```diff
@@ -1,232 +1,333 @@
...
+      <div class="col-12">
+        <h2>Get Ubuntu Pro on GCP now:</h2>
+      </div>
     </div>
-    <div class="col-3 p-divider__block">
-      <h3 class="p-heading--4"><a href="https://console.cloud.google.com/marketplace/product/ubuntu-os-pro-cloud/ubuntu-pro-xenial">16.04 LTS</a></h3>
+    <div class="row p-divider">
+      <div class="col-3 p-divider__block">
+        <h3 class="p-heading--4">
+          <a href="https://console.cloud.google.com/marketplace/product/canonical-public/ubuntu-pro-2204-jammy">22.04 LTS</a>
```
Comment (petesfrench) [inline templates/gcp/pro.html:312]: Changed

File: templates/gcp/pro.html [hunk 5]
```diff
@@ -1,232 +1,333 @@
...
+    <div class="row u-equal-height">
+      <div class="col-7 col-medium-4">
+        <h1>Ubuntu Pro for Google Cloud</h1>
+        <p class="p-heading--3">For production. For professionals.</p>
+        <p>
+          Ubuntu Pro for GCP is a premium image delivering the most comprehensive security and GCP compliance features. Ubuntu Pro is suitable for small and large-scale Linux enterprise operations offering pay-as-you-go billing on your existing GCP invoice.
+        </p>
+        <h3 class="p-heading--4">Ubuntu Pro 24.04 LTS</h3>
+        <p>
+          Delivers additional layer of security and compliance services and security patches covering a wider range of packages, until 2032.
```
Comment (akbarkz) [inline templates/gcp/pro.html:24]: There should be a change on this line too

**Review by akbarkz — APPROVED** [review]

**Pull request merged**